met-api/src/met_api/models/widget_item.py

```diff
@@ -66,3 +84,10 @@ def creat_all_widget_items(cls, widgets: list) -> list[WidgetItem]:
         db.session.add_all(new_widgets)
         db.session.commit()
         return new_widgets
+
+    @classmethod
+    def update_widget_items_bulk(cls, update_mappings: list) -> list[WidgetItem]:
+        """Save widget items sorting."""
+        db.session.bulk_update_mappings(WidgetItem, update_mappings)
+        db.session.commit()
```

Comment:
hm..do we commit in model all the time? we have to revisit since this will commit everything..
I think committing within the service layer will be better , since service might have related transactional calls

Reply:
Okay, noted.